e2e/playwright/info-drawer/src/tests/library-properties.spec.ts

```diff
@@ -0,0 +1,270 @@
+    await myLibrariesPage.libraryDetails.descriptionField.fill(newDesc);
+    await expect(myLibrariesPage.libraryDetails.updateButton).toBeEnabled();
+
+    await myLibrariesPage.libraryDetails.cancelButton.click();
+    expect(await myLibrariesPage.dataTable.isItemPresent(newName)).toBe(false);
+    expect(await myLibrariesPage.dataTable.isItemPresent(site.name)).toBe(true);
+    await expect(myLibrariesPage.libraryDetails.infoDrawerPanel).toBeVisible();
+  });
+
+  test('[C289341] Warning appears when editing the name of the library by entering an existing name', async ({ myLibrariesPage }) => {
+    queriesApi = await QueriesApi.initialize(username, username);
+    await queriesApi.waitForSites(site.name, { expect: 1 });
+    await myLibrariesPage.dataTable.getRowByName(siteDup).click();
+    await myLibrariesPage.acaHeader.viewDetails.click();
+    await expect(myLibrariesPage.libraryDetails.infoDrawerPanel).toBeVisible();
+    await expect(myLibrariesPage.libraryDetails.editButton).toBeVisible();
+
+    await myLibrariesPage.libraryDetails.editButton.click();
+    await myLibrariesPage.libraryDetails.nameField.fill(site.name);
+    await expect(myLibrariesPage.libraryDetails.hintMessage).toBeVisible();
+    expect(await myLibrariesPage.libraryDetails.hintMessage.textContent()).toEqual('Library name already in use');
+  });
+
+  test('[C289342] Site name too long', async ({ myLibrariesPage }) => {
+    await myLibrariesPage.dataTable.getRowByName(site.name).click();
+    await myLibrariesPage.acaHeader.viewDetails.click();
+    await expect(myLibrariesPage.libraryDetails.infoDrawerPanel).toBeVisible();
+
+    await myLibrariesPage.libraryDetails.editButton.click();
+    await myLibrariesPage.libraryDetails.nameField.fill(Utils.string257Long);
+    await expect(myLibrariesPage.libraryDetails.errorMessage).toBeVisible();
+    expect((await myLibrariesPage.libraryDetails.errorMessage.textContent()).trim()).toEqual('Use 256 characters or less for title');
+    await expect(myLibrariesPage.libraryDetails.updateButton).not.toBeEnabled();
+  });
+
+  test('[C289343] Site description too long', async ({ myLibrariesPage }) => {
+    await myLibrariesPage.dataTable.getRowByName(site.name).click();
+    await myLibrariesPage.acaHeader.viewDetails.click();
+    await expect(myLibrariesPage.libraryDetails.infoDrawerPanel).toBeVisible();
+
+    await myLibrariesPage.libraryDetails.editButton.click();
+    await myLibrariesPage.libraryDetails.descriptionField.fill(Utils.string513Long);
+    await expect(myLibrariesPage.libraryDetails.errorMessage).toBeVisible();
+    expect((await myLibrariesPage.libraryDetails.errorMessage.textContent()).trim()).toEqual('Use 512 characters or less for description');
+    await expect(myLibrariesPage.libraryDetails.updateButton).not.toBeEnabled();
+  });
+});
+
+test.describe('Non manager', () => {
+  let sitesApi: SitesApi;
+  const username = `user1-${Utils.random()}`;
```

Comment:
could you please keep separately consts and variables created by let? (you can separate them with empty line)